src/components/styles/PageSection.styled.ts

```diff
@@ -13,34 +13,51 @@ const fadeInUp = keyframes`
     }
 `
 
-export const MainContainer = styled.div<{ hasImage: boolean }>`
-  margin: 75px 0;
+export const MainContainer = styled.div<{ hasImage: boolean; mayNeedReverse?: boolean }>`
```

Comment:
mayNeedReverse seems a bit unclear? it's to reverse the order for mobile view?

Reply:
207942e